Formula/tbb.rb

```diff
@@ -32,6 +33,10 @@ def install
       system "cmake", "..", *args
       system "make"
       system "make", "install"
+      system "make", "clean"
+      system "cmake", "..", *args, "-DBUILD_SHARED_LIBS=OFF"
+      system "make"
+      lib.install Dir["**/*.a"]
```

Comment:
How much more complicated would it be to be more specific here?

Reply:
The full path looks like this:

```
./oneTBB-2021.5.0/build/appleclang_12.0_cxx11_64_release/libtbb.a
./oneTBB-2021.5.0/build/appleclang_12.0_cxx11_64_release/libtbbmalloc.a
```